tools/stimulus/types-RC.yaml

```diff
@@ -58,6 +58,10 @@ INT:
         IN: |-
             IGRAPH_R_CHECK_INT(%I%);
             %C% = INTEGER(%I%)[0];
+    OUTCONV:
+        OUT: |-
+            PROTECT(%I%=NEW_NUMERIC(1));
+            REAL(%I%)[0]=(double) %C%;
```

Comment:
It's fine to work with `INTEGER()` here too, as the returned numbers will be very small.

In the future, can you please open a separate PR in such cases?